package.json

```diff
@@ -51,10 +51,12 @@
     "@ethereumjs/vm": "5.3.1"
   },
   "dependencies": {
+    "@eth-optimism/core-utils": "^0.5.1",
```

Comment:
in OP code, `remove0x` does not handle negative numbers, they will start with `-0x` and inside the fn, you look for something that starts with `0x`.
It doesn't really matter know because I did not need to use it in order to convert numbers to hex

Reply:
I would prefer not to use an external library for things that are so small, because it will make slower the development in case we need to change something there.
Anyway, if you still want to, let's use it everywhere and delete those functions from utils

Reply:
Deleted in https://github.com/defi-wonderland/smock/pull/25